cobra/cmd/init_test.go

```diff
@@ -12,24 +12,22 @@ func TestGoldenInitCmd(t *testing.T) {
 	wd, _ := os.Getwd()
 	project := &Project{
 		AbsolutePath: fmt.Sprintf("%s/testproject", wd),
-		PkgName:      "github.com/spf13/testproject",
 		Legal:        getLicense(),
 		Copyright:    copyrightLine(),
-		Viper:        true,
-		AppName:      "testproject",
+
+		// required to init
```

Comment:
I think `AbsolutePath` is also required.  I'm sure others could be as well.  Pls remove comment and group separation.  Unless, that is, there is a really good reason that I'm not seeing.

Reply:
I just copied it from `add_test.go` so that the similarity between sources was easier to spot. Nonetheless, I added a helper function now (`getProject`) that is used by both `TestGoldenInitCmd` and `TestGoldenAddCmd`.